aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/service/SetupService.java

```diff
@@ -0,0 +1,194 @@
+package aero.minova.core.application.system.service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+import aero.minova.core.application.system.controller.SqlProcedureController;
+import aero.minova.core.application.system.domain.SqlProcedureResult;
+import aero.minova.core.application.system.sql.SystemDatabase;
+import lombok.val;
+
+@Service
+public class SetupService {
+
+	@Autowired
+	public SystemDatabase database;
+
+	@Autowired
+	public FilesService service;
+
+	@Autowired
+	SqlProcedureController spc;
+
+	Logger logger = LoggerFactory.getLogger(SetupService.class);
+
+	/**
+	 * Fügt die Erweiterung hinzu.
+	 */
+	private void setup() {
+		// Fügt Extension hinzu.
+		spc.registerExctension("setup", inputTable -> {
+			try {
+				SqlProcedureResult result = new SqlProcedureResult();
+				Path dependencyList = service.getSystemFolder().resolve("setup").resolve("dependencyList.txt");
+				if (dependencyList.toFile().exists()) {
+					List<String> procedures = readSetups(Files.readString(service.getSystemFolder().resolve("setup").resolve("dependencyList.txt")));
+					runDependencyProcedures(procedures);
+				} else {
+					throw new NoSuchFileException("No dependencyList.txt found!");
+				}
+				return new ResponseEntity(result, HttpStatus.ACCEPTED);
+			} catch (Exception e) {
+				throw new RuntimeException(e);
+			}
+		});
+	}
+
+	/**
+	 * Die Methode überspringt die erste Zeile (bis nach \n) und schneidet alles ab dem String "The following files have NOT been resolved:" ab. Danach wird
+	 * jede Zeile als ein Eintrag in der Rückgabe-Liste gesetzt. Dabei wird noch die Endung :jar entfernt und alle : durch . ersetzt.
+	 * 
+	 * @param arg
+	 *            Ein String, in welchem die benötigten Dependencies stehen.
+	 * @return Gibt eine Liste an String zurück, welche die benötigten Prozedur-Dateinamen beinhalten: prozedur.sql.
+	 */
+	public List<String> parseDependencyList(String arg) {
+		List<String> dependencies = new ArrayList<>();
```

Comment:
Kann man in die Zeile 82 einbringen.
Dadurch werden die Zeilen minimiert in denen die Variable verwendet wird.
Dadurch wird es nachvollziebarer.